extensions/cornerstone/src/services/ViewportService/CornerstoneViewportService.ts

```diff
@@ -863,7 +863,7 @@ class CornerstoneViewportService extends PubSubService implements IViewportServi
     });
   }
 
-  private _handleOverlays(viewport: Types.IStackViewport | Types.IVolumeViewport) {
+  private _processOverlaysForViewport(viewport: Types.IStackViewport | Types.IVolumeViewport) {
```

Comment:
Overlays has an awful lot of meetings - is this really about RT/Seg - if there is a better name for that sort of thing it would be helpful to use it.

Reply:
What about processExtraDIsplaySetsForViewport